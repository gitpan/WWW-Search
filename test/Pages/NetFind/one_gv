http://netfind.aol.com/dirsearch.adp?query=Zuckus
