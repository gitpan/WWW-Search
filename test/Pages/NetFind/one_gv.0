
<HTML>
<HEAD>
  <TITLE>AOL Search Preview : Search Results for Zuckus
</TITLE>
  <LINK REL=stylesheet TYPE=text/css HREF=/gr/styles.css>

</HEAD>
<BODY BGCOLOR="#ffffff" TEXT="#000000" LINK="#0000cc" VLINK="#000066" ALINK=#000099>
<!--- BEGIN NAV BAR ---> 
<CENTER>
<TABLE WIDTH=585 CELLPADDING=0 CELLSPACING=0 BORDER=0 ALIGN=CENTER>
        <TR>
                <TD WIDTH=119 BGCOLOR="#FFFFFF" ALIGN="LEFT"><A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/"><IMG SRC="http://www.aol.com/gr/mega_aol_banner.gif" WIDTH=119 HEIGHT=26 BORDER=0 ALIGN="LEFT"></A></TD>
                <TD WIDTH=466 BGCOLOR="#CC0000" VALIGN=MIDDLE ALIGN="CENTER"><NOBR>&nbsp;<FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/netfind/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Search</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/webcenters/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Web Centers</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/shopping/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Shopping</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/community/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Community</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/downloadaol.adp" STYLE="COLOR:#FFCC66"><FONT COLOR="#FFCC66" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Download AOL</B></FONT></A></FONT>&nbsp;</NOBR></TD>
        </TR>
        <TR>
                <TD WIDTH=585 HEIGHT=12 COLSPAN=2><HR SIZE=1></TD>
        </TR>
        <TR>
                <TD WIDTH=585 HEIGHT=12 COLSPAN=2>You are here: <A HREF=http://dynamic.aol.com/cgi/redir?http://www.aol.com/home.html>Home</A> &gt; AOL.COM Search</TD>
        </TR>
</TABLE>
</CENTER>

<!--- END NAV BAR --->

<P ALIGN=Center>
<A HREF=http://ads.web.aol.com/link/2014570/aol><img alt="Check Out Our Advertiser! ClickHere." border=0 height=60 width=468 src="http://ads.web.aol.com/image/2014570/aol?search=Zuckus"></A>

<P ALIGN=Center>
<CENTER>
  <TABLE WIDTH=585 BGCOLOR=#000066 BORDER="0" CELLSPACING="0" CELLPADDING="3"
  ALIGN="Center">
    <TR>
      <TD ROWSPAN=3 WIDTH=200 ALIGN=CENTER VALIGN=TOP><A HREF="index.adp"><IMG SRC="/gr/logo_com.gif"

	    WIDTH="198" HEIGHT="48" BORDER="0"></A></TD>
      <TD WIDTH=385><FONT COLOR=#ffffff>Enter search words below and then click
	Search.</FONT></TD>
    </TR>
<FORM ACTION="dirsearch.adp">
    <TR>
      <TD>
	  <TABLE BORDER="0" CELLSPACING="0" CELLPADDING="0">
	    <TR>
	      <TD>
		<INPUT TYPE=text NAME=query SIZE=25 VALUE="Zuckus">
</TD>
	      <TD>
		<IMG SRC=/gr/pixel.gif WIDTH=3 HEIGHT=10 BORDER=0><INPUT TYPE=submit VALUE="Search!"></TD>
	    </TR>
	  </TABLE>
      </TD>
    </TR>
	</FORM>
    <TR>
      <TD><A HREF="index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Home</FONT></A>
	<FONT COLOR="#ffffff">|</FONT>


	<A HREF="refine.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Search
	Options</FONT></A> <FONT COLOR="#ffffff">|</FONT>
	<A HREF="/help/index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Help</FONT></A></TD>
    </TR>
  </TABLE>
</CENTER>
<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR> 
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=565 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="10"
	    BORDER="0"></TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=440 VALIGN=TOP>
<TABLE CELLPADDING=4 CELLSPACING=0 BORDER=0>
<TR>
<TD>
<!-- RESULTS -->
</DIV>
<P>
<P>
<TABLE CELLPADDING=0 CELLSPACING=0 WIDTH=430 BORDER=0>
<TR>
<TD WIDTH=200><FONT COLOR=#990000 STYLE="font-family: Arial, helv, helvetica;  font-weight:bold; font-size:14px">MATCHING WEB PAGES</FONT></TD><TD WIDTH=50>&nbsp;</TD><TD WIDTH=130 ALIGN=CENTER>(<B>1</B> thru <B>7</B> of <B>7</B>)</TD><TD WIDTH=50 ALIGN=CENTER></TD></TR></TABLE>
<DIV ALIGN=left STYLE="font-size:10px; font-weight:light">
Some of the following results are from the World Wide Web and may contain objectionable material that AOL does not endorse.</DIV>
<P>		
<DIV STYLE="font-weight:normal">		<FONT COLOR=#990000 STYLE="font-weight:bold"><B>99%</B></FONT> <A HREF="http://www.uoe.ch/kennerfreezeframefigures.htm " STYLE="font-weight:bold"><B>
		kennerfreezeframefigures </B></A><BR>Freeze Frame Figures 8D8 Available 24.95 CHF ORDER NOW Admiral Ackbar Available 19.95 CHF ORDER NOW AT-AT Driver Available 29.95 CHF ORDER NOW AT-ST Driver Available 29.95 CHF ORDER NOW Ben Kenobi (Obi-Wan) Available 19.95 CHF ORDER NOW  Biggs.. <BR><I>http://www.uoe.ch/kennerfreezeframefigures.ht</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>98%</B></FONT> <A HREF="http://www.thepetchannel.com/News/disableriders.html " STYLE="font-weight:bold"><B>
		The Pet Channel - The Complete Online Resource for Pet Owners and Lovers </B></A><BR>Physically Challenged Riders To Compete In World Equestrian Games Fourth Level Dressage is a daunting challenge, even for able-bodied athletes. On September 13, Judy Nichols, a rider from California with severe nerve damage will show at  Fourth.. <BR><I>http://www.thepetchannel.com/News/disableride</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>95%</B></FONT> <A HREF="http://www.umcsc.org/users/jedi/ " STYLE="font-weight:bold"><B>
		Jedi's Corner </B></A><BR>Last Updated on June 7th, 1997 Jedi's Corner Latest Information on Star Wars Novels, Movies, and Kenner's POTF2. This Star Wars WebRing site is owned by David Connell (JediMasterLuke). Click for the [ Next Page | Skip It | Next 5 ] Want to join  the <BR><I>http://www.umcsc.org/users/jedi/ </I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>93%</B></FONT> <A HREF="http://rubylane.com/enter/toys/vintage/other/80.36.19.html " STYLE="font-weight:bold"><B>
		Ruby Lane Catalog for toys vintage other, page 1 </B></A><BR>Stroll down Ruby Lane to find the antiques and collectibles you've been searching for!  Whether you like fine antiques, paintings and art, or beanie babies, our search engine lists hundreds of thousands of items for sale at major auction and  shoppi <BR><I>http://rubylane.com/enter/toys/vintage/other/</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>66%</B></FONT> <A HREF="http://www.amazon.com/exec/obidos/ISBN=0553568167/002-0918377-0726245 " STYLE="font-weight:bold"><B>
		Amazon.com: A Glance: Star Wars : Tales of the Bounty Hunters </B></A><BR>at a glance reviews customer comments if you like this book.. read an excerpt related articles Keyword Search Books All Products Full search: Books, Music, Video, Toys or Electronics Visit the Amazon.com Star Wars Store! Star Wars : Tales of  the.. <BR><I>http://www.amazon.com/exec/obidos/ISBN=055356</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>62%</B></FONT> <A HREF="http://www.cs.bilkent.edu.tr/~david/derya/storys3/story1911.htm " STYLE="font-weight:bold"><B>
		The Young Writer's Club - Member's Stories No:1911 </B></A><BR>The Young Writer's Club Member's Stories No: 1911 Space Battle 2: Thresher 14 by 003, age 11 February 15, 2060 Greg Mueller here. Remember the 'Space Battle' I told you about not so long ago? There was another one about a year ago. It was  much.. <BR><I>http://www.cs.bilkent.edu.tr/~david/derya/sto</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>61%</B></FONT> <A HREF="http://www.comic-cafe.de/comiccafe/comic-cgis/gaestebuch.html " STYLE="font-weight:bold"><B>
		Gästebuch </B></A><BR>Gästebuch Hier können Sie sich eintragen. Tim Kusnik schrieb am 23.07.1999: Ich suche Spawn begeisterte di mit mir in Kontakt treten wollen Roman schrieb am 06.07.1999: ich suche illustrierte klassiker, zustand fast aber nicht aber preis ist  egal. <BR><I>http://www.comic-cafe.de/comiccafe/comic-cgis</I>
		<P></DIV><P>

<!-- END RESULTS -->
</TD>
</TR>
</TABLE>
</TD>
     
<TD WIDTH=2 BGCOLOR=#000066><IMG SRC=/gr/pixel.gif WIDTH=2 HEIGHT=1 BORDER=0></TD>
<TD WIDTH=123 VALIGN=TOP BGCOLOR=#000066>
<!-- SIDEBAR -->

<TABLE HEIGHT=100% WIDTH=100% BORDER=0 CELLSPACING=0 CELLPADDING=0>
   <TR>
    <TD VALIGN=TOP><TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0>
	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>
	<TR>
	  <TD>
<TABLE CELLPADDING=6 CELLSPACING=0 WIDTH=100% BORDER=0 BGCOLOR=#ccffcc>
<TR>
<TD ALIGN=CENTER><B>Search books:</B><BR>
<B>
<A HREF="http://dynamic.aol.com/cgi/redir?http://www.amazon.com/exec/obidos/aol-search/search=Zuckus">Zuckus</B><BR>
<A HREF="http://dynamic.aol.com/cgi/redir?http://www.amazon.com/exec/obidos/aol-search/search=Zuckus"><IMG SRC=/gr/amazonblk88x31.gif WIDTH=88 HEIGHT=31 BORDER=0></A>
</TD>
</TR>
</TABLE>

	  </TD>
	</TR>
<TR><TD><IMG SRC=/gr/pixel.gif WIDTH=1 HEIGHT=2 BORDER=0></TD></TR>  <TR>
    <TD>
</TD>
	</TR>
	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>

	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>
	<TR>
	  <TD><br>
<FONT COLOR=#ffffff><b>AOL Instant Messenger&#153;</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/aim/home.html"><FONT COLOR=#ffff00 CLASS=yellow>Instant Messenger&#153 2.0</FONT></a> is your free instant link to your friends, family.</FONT>
<p>


<FONT COLOR=#ffffff><b>Shop@AOL</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://shop.aol.com/"><FONT COLOR=#ffff00 CLASS=yellow>Great Bargains:</FONT></a> Digital shopping, right at your fingertips, from Shop@AOL.</FONT>
<p>


<FONT COLOR=#ffffff><b>Download AOL Now!</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/tryaol/"><FONT COLOR=#ffff00 CLASS=yellow>Try AOL:</FONT></a>. Free download. Try AOL now, and get 100 free hours, too.</FONT>
<p>


<FONT COLOR=#ffffff><b>Read Your News</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/mynews/home.adp"><FONT COLOR=#ffff00 CLASS=yellow>My News:</FONT></a> Get your news, from latest headlines to sports scores -all from My News.</FONT>
<p>


<FONT COLOR=#ffffff><b>Back to School '99</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/back2school/"><FONT COLOR=#ffff00 CLASS=yellow>School Days:</FONT></a> Chat with other students, get homework help, check out the latest fashions and more.</FONT>
<p>


<FONT COLOR=#ffffff><b>Web Centers</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/webcenters/realestate/home.adp"><FONT COLOR=#ffff00 CLASS=yellow>Real Estate Web Center:</FONT></a> From mortgages to rentals to moving, get all the answers to your real estate questions.</FONT>
<p>


<FONT COLOR=#ffffff><b>Your Sports</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/webcenters/sports/home.adp"><FONT COLOR=#ffff00 CLASS=yellow>Sports Web Center:</FONT></a> Catch up on the latest sports news, team reports, scores, and more.</FONT>
<p>






</TD>
	</TR>
      </TABLE>
    </TD>
  </TR>
	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>
  <TR>
    <TD VALIGN=BOTTOM ALIGN=CENTER>
<TABLE HEIGHT=100%>
<TR VALIGN=BOTTOM>
<TD VALIGN=BOTTOM>
      <A HREF="http://www.dmoz.org"><IMG SRC="/gr/dmoz.gif" WIDTH="88" HEIGHT="31"
	  BORDER="0"></A>
</TD>
</TR>
</TABLE>
</TD>
  </TR>
</TABLE>
<!-- END SIDEBAR -->
</TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
<CENTER>
<TABLE WIDTH=585 CELLPADDING=0 CELLSPACING=0 BORDER=0>
<TR>
<TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC=/gr/pixel.gif width=10 height=0 border=0></TD>
<TD WIDTH=565>
  <TABLE WIDTH=565 BGCOLOR=#000066 BORDER="0" CELLSPACING="0" CELLPADDING="3"
  ALIGN="Center">
    <TR>

      <TD ROWSPAN=3 WIDTH=200 ALIGN=CENTER VALIGN=TOP><A HREF="index.adp"><IMG SRC="/gr/logo_com.gif"

	    WIDTH="198" HEIGHT="48" BORDER="0"></A></TD>
      <TD WIDTH=385><FONT COLOR=#ffffff>Enter search words below and then click
	Search.</FONT></TD>
    </TR>
<FORM ACTION="dirsearch.adp">
    <TR>
      <TD>
	  <TABLE BORDER="0" CELLSPACING="0" CELLPADDING="0">
	    <TR>
	      <TD>
		<INPUT TYPE=text NAME=query SIZE=25 VALUE="Zuckus">
</TD>
	      <TD>
		<IMG SRC=/gr/pixel.gif WIDTH=3 HEIGHT=10 BORDER=0><INPUT TYPE=submit VALUE="Search!"></TD>
	    </TR>
	  </TABLE>
      </TD>
    </TR>
	</FORM>
    <TR>
      <TD><A HREF="index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Home</FONT></A>
	<FONT COLOR="#ffffff">|</FONT>
	<A HREF="refine.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Search
	Options</FONT></A> <FONT COLOR="#ffffff">|</FONT>
	<A HREF="/help/index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Help</FONT></A></TD>
    </TR>
  </TABLE>
</TD>
<TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC=/gr/pixel.gif width=10 height=0 border=0></TD>
</TR>
</TABLE>
</CENTER>

<!-- BOTTOM AD -->
<!--
<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=565 ALIGN=CENTER>
<BR>
<A HREF=http://ads.web.aol.com/link/2014570/aol><img alt="Check Out Our Advertiser! ClickHere." border=0 height=60 width=468 src="http://ads.web.aol.com/image/2014570/aol?search=Zuckus"></A>

<BR>&nbsp;
</TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
 -->
<!-- /END BOTTOM AD -->

<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=565><!-- ALSO SEARCH IN -->
<CENTER>
  <TABLE WIDTH=565 BORDER="0" CELLSPACING="3" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD><FONT COLOR="#003366"><B>Also Search In:</B></FONT></TD>
    </TR>
    <TR>
      <TD ALIGN=CENTER>
<A HREF="web.adp?query=Zuckus">Web Articles</A>
<B>&#149;</B>
<A HREF="http://hometown.aol.com/find.adp?cid=10000&resultOrder=1&queryToFlush=&query=Zuckus">Personal Home Pages</A> <B>&#149;</B>
      <A HREF="http://www.download.com/pc/list/1,339,0-a-0-0-1,00.html?tag=st.dl.fd.srch.list&queryType=quicksearch&search=Zuckus">Downloads</A>
      <B>&#149;</B>
      <A HREF="http://comptonsv3.web.aol.com/search/articlesearch?db1=A&db2=P&db3=F&db4=S&perpage=25&view=viewcomptons&userstring=Zuckus">Encyclopedias</A>
      <B>&#149;</B>
      <A HREF="http://www.deja.com/=aol/dnquery.xp?Search=Find%21&query=Zuckus">Newsgroups</A>
      <B>&#149;</B>
      <A HREF="http://www.thriveonline.com/cgi-bin/thriveall?vcs_personality=thrive%5fall&vcs_mode=search%5fdb&vcs_range=1%2d60&thriveterm=Zuckus">Health</A>

</TD>
    </TR>
    <TR>
      <TD><FONT COLOR="#003366"><B>Other Searches:</B></FONT></TD>
    </TR>
    <TR>
      <TD ALIGN=CENTER>
<A HREF="http://classifiedplus.aol.com/">ClassifiedPlus</A>
	<B>&#149;</B> <A HREF="http://www.moviefone.com/">Movie Times</A>
	<B>&#149;</B> <A HREF="http://www.aol.com/netfind/whitepages.adp">White
	Pages</A> <B>&#149;</B>
	<A HREF="http://www.aol.com/netfind/yellowpages.adp">Yellow Pages</A>

</TD>
    </TR>
  </TABLE>
</CENTER>
<!-- END ALSO SEARCH -->
<IMG SRC=/gr/pixel.gif WIDTH=565 HEIGHT=1 BORDER=0>
      </TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
    <TR>
      <TD COLSPAN=3 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="585" HEIGHT="10"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
<P>
<CENTER>
  <TABLE BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD ALIGN=CENTER><FONT STYLE="font-size:12px"><A HREF="/link_to_us.adp">Link to Us</A> | <A HREF="/about.adp">About AOL.COM
	Search</A> | <A HREF="/add.adp">Add Your Site</A> |
	<A HREF=http://mediaspace.web.aol.com>Advertise with Us</A><BR>Questions? Comments? <A HREF="/feedback.adp">Send us feedback.</A></FONT></TD>
    </TR>
    <TR>
      <TD><IMG SRC="/gr/pixel.gif" WIDTH="1" HEIGHT="5" BORDER="0"></TD>
    </TR>
    <TR>
      <TD ALIGN=CENTER><FONT STYLE="font-size:10px">Copyright &copy; 1999 America Online, Inc.<BR>
	All rights reserved. <A HREF="/copyright.adp">Legal Notices</A><BR>
<A HREF="http://www.aol.com/tryaol">Try AOL</A><BR>
<A HREF="http://www.aol.com/info/privacy.html">Privacy Policy</A></FONT></TD>
    </TR>
  </TABLE>
</CENTER>


</BODY></HTML>
