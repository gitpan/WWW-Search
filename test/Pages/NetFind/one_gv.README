This set of test-result pages was created on Mon Sep 20 09:20:57 1999
