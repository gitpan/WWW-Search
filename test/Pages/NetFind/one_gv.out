http://www.uoe.ch/kennerfreezeframefigures.htm 
http://www.thepetchannel.com/News/disableriders.html 
http://www.umcsc.org/users/jedi/ 
http://rubylane.com/enter/toys/vintage/other/80.36.19.html 
http://www.amazon.com/exec/obidos/ISBN=0553568167/002-0918377-0726245 
http://www.cs.bilkent.edu.tr/~david/derya/storys3/story1911.htm 
http://www.comic-cafe.de/comiccafe/comic-cgis/gaestebuch.html 
